src/compileTranslations.test.ts

```diff
@@ -2,6 +2,7 @@ import Vinyl from 'vinyl';
 
 import compileTranslations from './compileTranslations';
 import { PassThrough, Writable } from 'stream';
+import path from 'path';
```

Comment:
nit: Could you sort these imports? I know we're not super consistent about this, but it should be external, then a a newline, then local. Alphabetically sorted by filename within each.

Reply:
Sure!